packages/currency/test/currencyManager.test.ts

```diff
@@ -709,4 +709,46 @@ describe('CurrencyManager', () => {
       expect(path).toMatchObject([eur.hash, dai.hash.toLowerCase()]);
     });
   });
+
+  describe('Native and bridged USDC', () => {
+    const USDC_LIST = [
+      {
+        address: '0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174',
+        name: 'USD Coin (PoS)',
+        symbol: 'USDCe',
+        network: 'matic',
+        id: 'USDC-matic',
```

Comment:
can you give context why the ID wouldn't be `USDCe-matic` for this token?

Reply:
I added an elaborate comment above the describe block; there are many ways to explain this, I'm open to suggestions.

Reply:
Looks good!